actioncable/blade.yml

```diff
@@ -5,3 +5,9 @@ load_paths:
 
 logical_paths:
   - test.js
+
+build:
+  logical_paths:
+    - action_cable.js
+  path: lib/assets/compiled
+  js_compressor: uglifier
```

Comment:
@javan ideally we should include both, as done by others, and provide option to use either one. 
For me, it would be really important to have uncompressed version to check out the source, editor linking, source maps, etc. Again, this is because of how we develop right now, CS->JS. Other libs just include their source too, and 
provide dist versions of compressed/uncompressed versions, minus the noise like tests.

Right now we are on edge and ironing things out, I sometimes need to do manual exports for npm(not ideal), for making it work in some occasions, or some others might need to, to suit their export systems, commonjs, amd, etc.